wiki/Chat_Console/en.md

```diff
@@ -1,275 +1,106 @@
-</tr>
-<tr class="odd">
-<td><p>/reply or /r</p></td>
-<td><p>Reply to the last received private message.</p></td>
-<td><p>/r Do you know any good doctor?</p></td>
-<td><p>(At Amigo tab) [Previous comments]<br />
-John: I am sick at home.<br />
-Amigo: Really?<br />
-John: Do you know any good doctor?</p></td>
-</tr>
-<tr class="even">
-<td><p>/savelog</p></td>
-<td><p>Saves current chat tab to a text file.</p></td>
-<td><p>/savelog</p></td>
-<td><p>(A folder called &quot;Chat&quot; will be created at the osu! directory which will contain all the future chat tab saves)</p></td>
-</tr>
-<tr class="odd">
-<td><p>/watch <user></p></td>
-<td><p>Start spectating <user>.</p></td>
-<td><p>/watch Amigo</p></td>
-<td><ul>
-<li>Started spectating Amigo.<br />
-(When Amigo plays a beatmap that you have, you will spectate his play [after some buffering] with your name on the left of Amigo screen)</li>
-</ul></td>
-</tr>
-<tr class="even">
-<td><p>/nopm</p></td>
-<td><p>Toggle to allow private messages either from everyone or friends only</p></td>
-<td><p>/nopm</p></td>
-<td><p>(A <a href="Media:CC_nopm.jpg" title="wikilink">pop-up banner</a> will appear at the center detailing you are allowing everyone/friends only for private messages)</p></td>
-</tr>
-<tr class="odd">
-<td><p>/invite <user></p></td>
-<td><p>Invites <user> to the multiplayer room along with the link.</p></td>
-<td><p>/invite Nathanael</p></td>
-<td><ul>
-<li>Nathanael has been invited to the game</li>
-</ul></td>
-</tr>
-</tbody>
-</table>
-
-</center>
-/keys
------
-
-<center>
-<table style="width:80%;">
-<colgroup>
-<col width="12%" />
-<col width="68%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th><p>Keyboard keys</p></th>
-<th><p>Effect</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>PageUp / PageDown</p></td>
-<td><p>Scroll the chat window.You may also use the mouse-wheel.</p></td>
-</tr>
-<tr class="even">
-<td><p>Tab</p></td>
-<td><p>Auto-complete currently typing nickname.</p></td>
-</tr>
-<tr class="odd">
-<td><p>F8</p></td>
-<td><p>Toggle chat console.</p></td>
-</tr>
-<tr class="even">
-<td><p>F9</p></td>
-<td><p>Toggle extended chat console.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Ctrl + C/V</p></td>
-<td><p>Copy/Paste.</p></td>
-</tr>
-<tr class="even">
-<td><p>Alt + 0-9</p></td>
-<td><p>Switch to respective tab.</p></td>
-</tr>
-</tbody>
-</table>
-
-</center>
+## Commands list
+
+### /help
+
+| Command        | Effect                                                                                         | Example           | BanchoBot response                                                               | 
+|------------------|------------------------------------------------------------------------------------------------|--------------------|----------------------------------------------------------------------------------|
+| /addfriend [user]   | Add [user] to your friends list.      | /addfriend Amigo | You are now friends with Amigo. |
```

Comment:
- since `/addfriend [user]` is a command type of thing, use code `` `/addfriend [user]` ``
- since `[user]` is an argument type of thing, use code `` `[user]` ``
- since `/addfriend Amigo` is an example of using a command, use code `` `/addfriend Amigo` ``

please do the same below